homeassistant/components/homeassistant/strings.json

```diff
@@ -11,6 +11,10 @@
     "python_version": {
       "title": "Support for Python {current_python_version} is being removed",
       "description": "Support for running Home Assistant in the current used Python version {current_python_version} is deprecated and will be removed in Home Assistant {breaks_in_ha_version}. Please upgrade Python to {required_python_version} to prevent your Home Assistant instance from breaking."
+    },
+    "integration_key_no_support": {
+      "title": "This integration does not support YAML configuration",
+      "description": "The {domain} integration does not support configuration via YAML file. You may not notice any obvious issues with the integration, but the configuration settings defined in YAML are not actually applied. \n\n1. Please remove this integration from your YAML configuration file.\n\n2. Restart Home Assistant."
```

Comment:
Agree, I think that helps with the intent of the steps presented.

Otherwise, the issue message will have some weird step in the middle, that a user might not understand the reason "Why".

../Frenck